CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Enhancements
+
+- Add `metrics` prefix to the url of list instances and list targets endpoints (@marctc)
```

Comment:
Can we list the URLs that have changed here to make it easier for the user to understand what we're talking about?